### PR TITLE
Update abnormality queue and meltdowns

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About The Pull Request

This simply prepares the abnormality queue for manager's ability to pick abnormalities.
Additionally, HEs stop spawning when ALEPHs appear.
Meltdowns do not happen for ZAYIN and TETH abnormalities after some point into the game.

### CODE REVIEW

File: code/controllers/subsystem/abnormality_queue.dm [hunk 1]
```diff
@@ -15,21 +20,26 @@ SUBSYSTEM_DEF(abnormality_queue)
...
 	..()
 
 /datum/controller/subsystem/abnormality_queue/fire()
 	if(world.time >= (wait*3))
-		if(!(HE_LEVEL in available_levels))
-			available_levels += HE_LEVEL
-	if(world.time >= (wait*6))
 		if(ZAYIN_LEVEL in available_levels)
 			available_levels -= ZAYIN_LEVEL
+		if(!(HE_LEVEL in available_levels) && world.time < (wait*9))
```
Comment (dsmith328) [inline code/controllers/subsystem/abnormality_queue.dm:30]: Doesn't this mean that if it's BEFORE 45 minutes and HE arent in, they get added in?

Comment (EgorDinamit) [reply code/controllers/subsystem/abnormality_queue.dm:30]: Yes.

File: code/modules/mob/living/simple_animal/abnormality/zayin/wellcheers.dm [hunk 2]
```diff
@@ -59,9 +59,9 @@
 	return
 
 /mob/living/simple_animal/hostile/abnormality/wellcheers/work_chance(mob/living/carbon/human/user, chance)
-	return chance - success_counter
+	return chance - (success_counter * 2)
```
Comment (dsmith328) [inline code/modules/mob/living/simple_animal/abnormality/zayin/wellcheers.dm:62]: Holy shit the real change. Shadow Nerfing Wellcheers

Comment (SpaceJohn413) [reply code/modules/mob/living/simple_animal/abnormality/zayin/wellcheers.dm:62]: qliphoth overload on a zayin

File: code/controllers/subsystem/abnormality_queue.dm (hunk 1 above)
Comment (dsmith328) [reply code/controllers/subsystem/abnormality_queue.dm:30]: Neat

Comment (Kitsunemitsu) [reply code/controllers/subsystem/abnormality_queue.dm:30]: Not sure if we have enough WAWs and Alephs for this, I'd keep HEs but make them much less common

File: code/modules/mob/living/simple_animal/abnormality/zayin/wellcheers.dm (hunk 2 above)
Comment (Kitsunemitsu) [reply code/modules/mob/living/simple_animal/abnormality/zayin/wellcheers.dm:62]: Why are we nerfing wellcheers _again_

**Pull request merged**